### ISSUE TITLE
npm grunt-livereload - Using some pluggins for grunt but the livereload doesn't work

### ISSUE THREAD

**Issue by jessicamichel93** [npm/npm#7165]
Hi everyone!

First, I'm a newbie to grunt, I learned a couple days ago the base and used grunt for my new theme in wordpress.

Well everything works fine, one the last plugin the livereload does't work, can somebody please help me out?

If somebody know the bug, I'm very curious what it was, cause I can't fix it.

Thanks!

PS: If I call only the livereload function it gives some kind of error (check printscreen)

gruntfile.js

module.exports = function(grunt) {

	grunt.initConfig({
		pkg: grunt.file.readJSON('package.json'),
		
		watch: {
			sass: {
				options: { livereload: true },
				files: ['dev/sass/*.scss'],
				tasks: ['sass', 'cssmin'],
			}
		},

		
		sass: {
			dist:{
				files: {
					'build/css/style.css' : 'dev/sass/style.scss'
				}
			}
		},
		
		uglify: {
			options: {
				manage: false
			},
			my_target: {
				files: {
					'build/js/simply-purple.min.js': ['dev/js/*.js']
				}
			}
		},
		
		cssmin: {
			my_target: {
				files: [{
					expand: true,
					cwd: 'build/css/',
					src: ['*.css', '!*.min.css'],
					dest: 'build/css/',
					ext: '.min.css'
				}]
			}	
		},
		
			browserSync: {
			bsFiles: {
				src : 'assets/build/css/*.css'
			},
			options: {
				proxy: "localhost/jmwebdevelopment/",
				watchTask: true,
				livereload: true
			}
		},
		
		livereload: {
			options: {
				livereload: true
			}
		}
		

		
	});
	
	
	
	grunt.loadNpmTasks('grunt-contrib-watch');
	grunt.loadNpmTasks('grunt-contrib-sass');
	grunt.loadNpmTasks('grunt-contrib-uglify');
	grunt.loadNpmTasks('grunt-contrib-cssmin');
	grunt.registerTask('default', ['watch']);
	grunt.loadNpmTasks('grunt-browser-sync');
	grunt.loadNpmTasks('grunt-livereload');
	
	  // define default task
    grunt.registerTask('default', ["browserSync", "watch", "livereload"]);
};
![terminal](https://cloud.githubusercontent.com/assets/8256028/5800416/8aa625e0-9fde-11e4-8635-41a15c14842e.png)

**Comment by KenanY** [comment]
@jessicamichel93 Hello! This is the issue tracker for `npm` (the package manager), but your issue doesn't appear to be related to `npm` at all. As such, it might be hard getting help with anything `grunt`-related here. I recommend that you open an issue with the `grunt` plugin that you are having issues with :)

**Comment by jessicamichel93** [comment]
Hi, thank you, but how can I find the right direction for grunt issues?

**Comment by KenanY** [comment]
@jessicamichel93 Since it looks like your issue is with `grunt-livereload`, I'd check that repo first. The following command will open its GitHub repo in your browser:

``` bash
$ npm repo grunt-livereload
```

Which should open up https://github.com/ericclemmons/grunt-livereload. Hmm, looks like that module has been deprecated for over two years now. It would probably be in your best interest to refactor your `Gruntfile.js` to use [`grunt-contrib-watch`](https://github.com/gruntjs/grunt-contrib-watch) instead, which has [livereload support](https://github.com/gruntjs/grunt-contrib-watch#optionslivereload).

**Comment by KenanY** [comment]
Erm, I guess you're already using `grunt-contrib-watch`. Anyways, probably best to open an issue at their repo instead of here. The warning in your screenshot says you're missing some required configuration settings.

**Comment by jessicamichel93** [comment]
thanks for looking, yeah I also found out that there is missing some require configuration, but I don't know which one. 

seriously when I search for grunt livereload issues I found some stuff and dropped my issue here.. https://github.com/Crosscheck/drupal-theme-boilerplate/issues/51 
but know It's look like its in the drupal issues, like I sad GH and grunt is really new for me, can u maybe help me out? Where can I post it correctly haha?
